frameworks/Rust/rocket/rocket.dockerfile

```diff
@@ -1,18 +1,18 @@
-FROM rust:1.46.0-slim-buster
+FROM rust:1.55-slim-buster
 
 ENV DATABASE_URL=postgres://benchmarkdbuser:benchmarkdbpass@tfb-database/hello_world
```

Comment:
You may want to change this variable to match the one you use in main.rs

Reply:
Fixed